File: easybluetoothble/src/main/java/com/srise/easybluetoothble/bean/BaseMessage.java
package com.srise.easybluetoothble.bean;

import android.bluetooth.BluetoothDevice;
import android.bluetooth.BluetoothGattCharacteristic;

import com.srise.easybluetoothble.Interface.IMessageDataProcessFinish;

public class BaseMessage {
    private byte[] mMessageBody;
    private BluetoothDevice mDevice;
    private BluetoothGattCharacteristic mCharacteristic;
    private IMessageDataProcessFinish mFinishCallBack;

    public IMessageDataProcessFinish getFinishCallBack() {
        return mFinishCallBack;
    }

    public void setFinishCallBack(IMessageDataProcessFinish finishCallBack) {
        this.mFinishCallBack = finishCallBack;
    }

    public byte[] getMessageBody() {
        return mMessageBody;
    }

    public void setMessageBody(byte[] messageBody) {
        this.mMessageBody = messageBody;
    }

    public BluetoothDevice getDevice() {
        return mDevice;
    }

    public void setDevice(BluetoothDevice device) {
        this.mDevice = device;
    }

    public BluetoothGattCharacteristic getCharacteristic() {
        return mCharacteristic;
    }

    public void setCharacteristic(BluetoothGattCharacteristic characteristic) {
        this.mCharacteristic = characteristic;
    }
}

File: ktlionble/src/main/java/c/example/dell/ktlionble/bean/BaseMessage.kt
package c.example.dell.ktlionble.bean

import android.bluetooth.BluetoothDevice
import android.bluetooth.BluetoothGattCharacteristic
import c.example.dell.ktlionble.inter.IMessageDataProcessFinish

class BaseMessage {
    var mMessageBody :  ByteArray? = null
    var mDevice: BluetoothDevice? = null
    var mCharacteristic : BluetoothGattCharacteristic? = null
    var mFinishCallBack : IMessageDataProcessFinish? = null
}